src/Controller/Controller.php

```diff
@@ -267,14 +267,14 @@ public function components(?ComponentRegistry $components = null): ComponentRegi
     /**
      * Add a component to the controller's registry.
      *
-     * This method will also set the component to a property.
+     * After loading a component it will be be accessible as a property through Controller::__get().
```

Comment:
This is technically true, but do we really document the use of `__get()` itself? Can we call it an annotated property or something? Is there a standard term?

Reply:
I think a lot of people won't understand what's an "annotated property". It seems the standard term is "overloaded" or "dynamic" property https://www.php.net/manual/en/language.oop5.overloading.php.

Reply:
But the [RFC](https://wiki.php.net/rfc/deprecate_dynamic_properties) says

> It should be noted that properties accessed through __get()/__set() are not considered as “dynamic properties”.

so :man_shrugging: .

Reply:
Yea, I think that part of the RFC is misstated. It's just to clarify since the name of the RFC is deprecating dynamic properties.

Reply:
So should the text be changed to `..it will be be accessible as a dynamic property.`?

Reply:
I think it works but up to you.